url_creator.py

```diff
@@ -0,0 +1,6 @@
+from os import path
+
+AMAZON_BASE_URL = 'https://www.amazon.com/dp'
+
+def creat_product_url(product_id):
+    return path.join(AMAZON_BASE_URL, product_id)
```

Comment:
after changing the base URL, this should become
~~~~
return path.join(AMAZON_BASE_URL, 'dp', product_id)
~~~~